### PR TITLE
fix: remove outdated cert store from packages

### PR DESCRIPTION
**Pull request by Traphalet**
Fixes EEC-636

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

**Review by id — APPROVED** [review]

**Comment by zmstone** [comment]
if this is not for 4.3.22, please change base to main-v4.3

**Comment by zmstone** [comment]
> if this is not for 4.3.22, please change base to main-v4.3

nvm

**Review by zmstone — APPROVED** [review]

File: build [hunk 1]
```diff
@@ -61,9 +61,22 @@ log() {
...
+}
+
 make_rel() {
     # shellcheck disable=SC1010
-    ./rebar3 as "$PROFILE" do release,tar
+    ./rebar3 as "$PROFILE" do release
+    # delete outdated cert store
+    delete_unwanted_files _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
+    # shellcheck disable=SC1010
+    ./rebar3 as "$PROFILE" do tar
```
Comment (zmstone) [inline build:79]:
```suggestion
    ./rebar3 as "$PROFILE" release
    # delete outdated cert store
    delete_unwanted_files _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
    ./rebar3 as "$PROFILE" tar
```

File: build [hunk 2]
```diff
@@ -61,9 +61,22 @@ log() {
...
+    if [ -e "${1}" ]; then
+        log "Deleting file: ${1}"
+        rm -f "${1}"
+    else
+        log "Cannot delete file: ${1} -- file not found"
+    fi
+}
+
 make_rel() {
     # shellcheck disable=SC1010
```
Comment (zmstone) [inline build:74]:
```suggestion
```

Comment (Traphalet) [reply build:74]: shellcheck signals SC1010 for both `./rebar3 as "$PROFILE" do release` and `./rebar3 as "$PROFILE" do tar` - disabling in both places is needed

Comment (Traphalet) [reply build]: https://github.com/emqx/emqx/actions/runs/3419230186/jobs/5692489889#step:4:21

File: build [hunk 3]
```diff
@@ -61,9 +61,22 @@ log() {
     echo "===< $msg"
 }
 
+delete_unwanted_files() {
```
[the remark this answers is not in the document]
Comment (Traphalet) [reply build]: I used plural in the event some other things need to be wiped post release build :) Changed as suggested.

File: build (hunk 1 above)
Comment (Traphalet) [reply build]: Same as below, SC complains about both places where SC1010 is not suppressed

File: build [hunk 4]
```diff
@@ -61,9 +61,22 @@ log() {
...
+        rm -f "${1}"
+    else
+        log "Cannot delete file: ${1} -- file not found"
+    fi
+}
+
 make_rel() {
     # shellcheck disable=SC1010
-    ./rebar3 as "$PROFILE" do release,tar
+    ./rebar3 as "$PROFILE" do release
```
Comment (zmstone) [inline build:75]:
```suggestion
    ./rebar3 as "$PROFILE" release
```

File: build [hunk 5]
```diff
@@ -61,9 +61,22 @@ log() {
...
+    # delete outdated cert store
+    delete_unwanted_file _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
+    # shellcheck disable=SC1010
+    ./rebar3 as "$PROFILE" do tar
```
Comment (zmstone) [inline build:79]:
```suggestion
    ./rebar3 as "$PROFILE" tar
```

File: build [hunk 6]
```diff
@@ -61,9 +61,22 @@ log() {
...
+    fi
+}
+
 make_rel() {
     # shellcheck disable=SC1010
-    ./rebar3 as "$PROFILE" do release,tar
+    ./rebar3 as "$PROFILE" do release
+    # delete outdated cert store
+    delete_unwanted_file _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
+    # shellcheck disable=SC1010
```
Comment (zmstone) [inline build:78]:
```suggestion
```

File: build (hunk 2 above)
Comment (zmstone) [reply build:74]: remove the `do` should fix it.

File: build (hunk 6 above)
Comment (Traphalet) [reply build]: updated

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
Thank you for the fix.

**Pull request merged**